### PR TITLE
feat: update automatic tagging decision

### PR DESCRIPTION
**Pull request by rpenido**
Documenting the decision to move from using `openedx-filters` to `openedx-events`

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @rpenido!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


This is currently a draft pull request. When it is ready for our review and all tests are green, click "Ready for Review", or remove "WIP" from the title, as appropriate.

File: docs/decisions/0013-system-taxonomy-auto-tagging.rst [hunk 1]
```diff
@@ -4,25 +4,23 @@
...
-Use `openedx-filters`_ to call the auto tagging function after content creation/edition.
-
-Filters
+Events
 ~~~~~~~~
 
-It is necessary to create `filters`_ for each content for creation/edition: ``CourseCreation``, ``LibraryCreation``, etc.
-This filters will live on ``openedx-filters``.
+It is necessary to create `events`_ for each content for creation/edition: ``CourseCreation``, ``LibraryCreation``, etc.
+This events will live on ``openedx-events``.
```
Comment (pomegranited) [inline docs/decisions/0013-system-taxonomy-auto-tagging.rst:17]: Could you add a link here, and expand it [below](https://github.com/openedx/openedx-learning/pull/61/files#diff-2113051e4c045e58c5e9eed6c560e78598409cca5f91a830e601703a1256b5b3R45)?
```suggestion
This events will live in `openedx-events`_.
```

File: docs/decisions/0013-system-taxonomy-auto-tagging.rst [hunk 2]
```diff
@@ -31,10 +29,22 @@ Django Signals
...
-as a Django signal. This works for Django database models, but some of the content lives in Mongo, 
+as a Django signal. This works for Django database models, but some of the content lives in Mongo,
 outside of the Django models. Also, using openedx-filters is better in the edx context, but if there is
 other no-edX project that need to use ``openedx-tagging``, can use the Django Signals approach.
 
+
+openedx-filters
+~~~~~~~~~~~~~~~
+Use `openedx-filters`_ to create a `filter`_ call `the auto tagging pipeline`_ after content
+creation/edition. Although this approach works, there are more suitable options. Filters are
```
Comment (pomegranited) [inline docs/decisions/0013-system-taxonomy-auto-tagging.rst:40]: nits:
```suggestion
Use `openedx-filters`_ to create a `filter`_ which calls `the auto tagging pipeline`_ after content
creation/editing. Although this approach works, there are more suitable options. Filters are
```

**Review by pomegranited — APPROVED** [review]
Thanks for this PR @rpenido ! Couple of nits to fix on wording, but I agree it's the right way to go here.

@ormsbee Do you agree?

:+1:

- [x] I read through the code and rendered documentation
- [x] Commit structure follows [OEP-0051](https://github.com/edx/open-edx-proposals/pull/189)

**Review by ormsbee — APPROVED** [review]

**Comment by rpenido** [comment]
Fixing wording and squashing commits

**Comment by openedx-webhooks** [comment]
@rpenido 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/openedx-learning&entry.867055334=https://github.com/openedx/openedx-learning/pull/61&entry.1484655318=https://github.com/rpenido&entry.752974735=2023-06-29+12:29&entry.1917517419=2023-07-04+01:30&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**